packages/aws-cdk-lib/aws-rds/lib/cluster.ts

```diff
@@ -802,7 +831,7 @@ abstract class DatabaseClusterNew extends DatabaseClusterBase {
       throw new Error('`enablePerformanceInsights` disabled, but `performanceInsightRetention` or `performanceInsightEncryptionKey` was set');
     }
 
-    if (props.clusterScailabilityType === ClusterScailabilityType.LIMITLESS) {
+    if (props.clusterScalabilityType === ClusterScalabilityType.LIMITLESS || props.clusterScailabilityType === ClusterScailabilityType.LIMITLESS) {
```

Comment:
Also needs a check to ensure that you can't set both the old and new property at the same time. And that should be mentioned in the docstring as well.

Reply:
Thanks, will make this change.